LoggingKata.Test/TacoParserTests.cs

```diff
@@ -1,29 +1,49 @@
 using System;
 using Xunit;
+using System.Collections.Generic;
 
 namespace LoggingKata.Test
 {
     public class TacoParserTests
     {
         [Fact]
         public void ShouldDoSomething()
-        {
-            // TODO: Complete Something, if anything
+        { 
+            Console.WriteLine("Hello Chip!");
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("34.8831, -84.293899, Taco Bell Blue Ridg")]
+        [InlineData("30.39371, -87.68332, Taco Bell Fole")]
+        [InlineData("33.61915,-84.243415,Taco Bell Ellenwoo")]
+        [InlineData("1, 2, Example")]
         public void ShouldParse(string str)
         {
-            // TODO: Complete Should Parse
+            // Arrange
+            var TacoParser = new TacoParser();
+
+            // Act
+            ITrackable answer = TacoParser.Parse(str);
+
+            // Assert
+            Assert.NotNull(answer);
         }
 
         [Theory]
         [InlineData(null)]
         [InlineData("")]
+        [InlineData("15Four65")]
+        [InlineData("sixteen, happyBirthDayToMe, sn00pd0ggyd0g")]

```

Comment:
There could be a few more (potentially a lot depending on how good we want the Parse method to be) tests for things that should fail. Perhaps:
```CS
"10, ten, Taco Bell"
"ten, 10, Taco Bell"
"10, 10"
"10, 10, "
"10,,Taco Bell"
",10, Taco Bell"
", , Taco Bell"
"10, Taco Bell, 10"
"Taco Bell, 10, 10"
"91, 10, Taco Bell" // above max latitude
"-91, 10, Taco Bell" // below min latitude
"10, 181, Taco Bell" // above max longitude
"10, -181, Taco Bell" // below min longitude
```
etc.